api/handler/v1/runtime.go

```diff
@@ -81,10 +80,11 @@ type RuntimeServiceServer struct {
 	Now              func() time.Time
 
 	pb.UnimplementedRuntimeServiceServer
+	l log.Logger
 }
 
 func (sv *RuntimeServiceServer) Version(ctx context.Context, version *pb.VersionRequest) (*pb.VersionResponse, error) {
-	log.I(fmt.Printf("client with version %s requested for ping ", version.Client))
+	sv.l.Info("client with version %s requested for ping ", version.Client)
```

Comment:
I don't think this kind of notation will work. You should do something like
```
sv.l.Info("client requested for ping ", "version", version.Client)
```
So the format should be, 
"a long string that explains the log" then "key", then its "value"